build.gradle

```diff
@@ -25,6 +25,7 @@ project.ext {
     riffVersion = '2.4.6'
     jacksonVersion = '2.9.6'
     jettyVersion = '9.4.12.v20180830'
+    bouncyCastle = '1.68'
```

Comment:
Can we stay consistent to the naming and use something like `bouncyCastleVersion` ?

Reply:
yep, I somehow missed that